test/_unit.js

```diff
@@ -16,6 +16,19 @@ async function loadParser () {
 suite('Lexer', () => {
   beforeEach(async () => await loadParser());
 
+  test('esbuild hint style', () => {
+    var { exports, reexports } = parse(`
+      0 && module.exports = {a, b, c} && __exportStar(require('fs'));
```

Comment:
This is a syntax error actually. So the real code would have to look like this instead:

```js
0 && (module.exports = {a, b, c}) && __exportStar(require('fs'));
```

Might want to consider changing the test to look like that instead, or in addition to this test.

Reply:
Thanks, fixed up in https://github.com/guybedford/cjs-module-lexer/commit/a352cbc1b0b3b0cc165008ae9ed4b3e9ec1df799.